### DISCUSSION TITLE
Problem with AndroidLibV2rayLite go.mod

### DISCUSSION THREAD

**Discussion by ghost** [2dust/v2rayNG#1034] [Q&A]
the previous version works
github.com/2dust/AndroidLibV2rayLite v0.0.0-20201207002626-7c88a459716a

but this version gives me error
github.com/2dust/AndroidLibV2rayLite v0.0.0-20210313024657-2c3968714c28

**go mod download github.com/2dust/AndroidLibV2rayLite
go: github.com/2dust/AndroidLibV2rayLite@v0.0.0-20210313024657-2c3968714c28: parsing go.mod: go.mod:6:2: require github.com/v2fly/v2ray-core/v4: version "master" invalid: must be of the form v1.2.3**

I tried to use replace still does not work

replace github.com/v2fly/v2ray-core/v4 master => github.com/v2fly/v2ray-core/v4 v4.38.2

**Comment by rurirei** [comment]
i'll do NOT agreed to uses the lib as a dependencies, you should rework on that to meet your needs.
